apps/marketing/src/components/(marketing)/single-player-mode/create-single-player-document.action.ts

```diff
@@ -214,11 +213,14 @@ const mapField = (
   field: TCreateSinglePlayerDocumentSchema['fields'][number],
   signer: TCreateSinglePlayerDocumentSchema['signer'],
 ) => {
-  const customText = match(field.type)
-    .with(FieldType.DATE, () => DateTime.now().toFormat('yyyy-MM-dd hh:mm a'))
-    .with(FieldType.EMAIL, () => signer.email)
-    .with(FieldType.NAME, () => signer.name)
-    .otherwise(() => '');
+  const customText =
+    field.type === FieldType.DATE
+      ? DateTime.now().toFormat('yyyy-MM-dd hh:mm a')
+      : field.type === FieldType.EMAIL
+      ? signer.email
+      : field.type === FieldType.NAME
+      ? signer.name
+      : '';
```

Comment:
Sorry but we have decided to use `match` for the majority of use cases such as this in our codebase.

Nested ternaries are rather unreadable, messy and more complex than `match`.

Note: If `match` did not exist in this scenario a switch case would be preferred.

Reply:
Yeah sorry, if you compare match with the ternary here it becomes pretty clear how powerful it is from a readability perspective 😇

Reply:
@Mythie Makes sense, I guess this aligns with another ternary change which we had discussed wrt another issue, I will revert this :)